permissions/plugin-cloverphp.yml

```diff
@@ -6,4 +6,12 @@ issues:
 paths:
 - "org/jenkins-ci/plugins/cloverphp"
 developers:
-- "sogabe"
\ No newline at end of file
+- "sogabe"
+- "alecharp"
+- "batmat"
+- "bitwiseman"
+- "carroll"
+- "mramonleon"
+- "olamy"
+- "rsandell"
+- "teilo"
```

Comment:
GitHub accounts for these are:

@alecharp, @bitwiseman, @car-roll, @MRamonLeon, @olamy , @rsandell, @jtnord